### PR TITLE
86b1u19ry

### PR DESCRIPTION
**Pull request by ThilanN**
added the reject accept to the mentor pop up ui

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -28,6 +27,7 @@
     "dotenv-cli": "^7.4.2",
     "lucide-react": "^0.436.0",
     "next": "14.2.5",
+    "npm": "^10.8.2",
```
Comment (bsaranga) [inline server/package.json]: this should not be a dependency

**Review by bsaranga — CHANGES_REQUESTED** [review]
Fix merge conflicts and push again

File: server/src/components/MentorPopUp.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    </AlertDialog>
+  );
+};
+
+const sampleMentorDetails = {
+  selectedDate: "2024-09-05",
+  workingHours: "3 hours",
+  studentActivity: "Completed the project proposal and submitted it for review.",
+  review: ""
+};
```
Comment (bsaranga) [inline server/src/components/MentorPopUp.tsx:99]: @thlurte once this is merged, remove this and use db integration.

File: server/package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "npm": "10.8.2"
   },
   "dependencies": {
+    "@ant-design/nextjs-registry": "^1.0.1",
```
Comment (bsaranga) [inline server/package.json]: Remove this, we're not using antd anymore

**Review by bsaranga — CHANGES_REQUESTED** [review]

File: server/package.json (hunk 3 above)
Comment (ThilanN) [reply server/package.json]: removed

**Review by bsaranga — APPROVED** [review]

**Pull request merged**